test/unit/organisation_content_presenter_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class OrganisationContentPresenterTest < ActiveSupport::TestCase
+  should "provide a CSV export of business support schemes" do
+    document = FactoryGirl.create(:artefact,
+                                  name: "Important document",
+                                  kind: "answer",
+                                  need_ids: %w(123456 123321 654321))
+    FactoryGirl.create(:edition,
+                       panopticon_id: document.id)
+
+    csv = OrganisationContentPresenter.new(
+      Artefact.where(owning_app: "publisher").not_in(state: ["archived"])
+    ).to_csv
+    data = CSV.parse(csv, headers: true)
+
+    assert_equal "Important document", data[0]["Name"]
+    assert_equal "answer", data[0]["Format"]
+    assert_equal "123456,123321,654321", data[0]["Need ids"]
+  end
+
+  should "handle artefacts without editions" do
+    FactoryGirl.create(:artefact,
+                       name: "Important document",
+                       need_ids: ["123456"])
+
+    csv = OrganisationContentPresenter.new(
+      Artefact.where(owning_app: "publisher").not_in(state: ["archived"])
+    ).to_csv
+    data = CSV.parse(csv, headers: true)
+
+    assert_equal "Important document", data[0]["Name"]
+  end
+
+  should "call publishing_api 3 times" do
+    Services.publishing_api.expects(:get_expanded_links).times(3)
+    document = FactoryGirl.create(:artefact,
+                                  name: "Important document",
+                                  need_ids: ["123456"])
+    FactoryGirl.create(:edition,
+                       panopticon_id: document.id)
+
+    OrganisationContentPresenter.new(
+      Artefact.where(owning_app: "publisher").not_in(state: ["archived"])
+    ).to_csv
+  end
+
+  should "retrieve the information needed from the JSON response" do
+    JSON = { 'expanded_links' => { 'organisations' => ['title' => 'HMRC'] } }.freeze
+    str = OrganisationContentPresenter.new.send(:join_strings, JSON, 'organisations', 'title', /.^/)
```

Comment:
It's not usually a good idea to test private methods directly, but to test the outcome of them via the public methods (not least because public methods tend to be more stable than private ones, and it gives you more flexibility with your implementation - a private method is often a candidate for extraction somewhere else during a refactor for example).  

I'd prefer to see a test similar to the first one that pushes an artefact through the presenter and checks the values in the outputted CSV data.  This time though also stubs out the publishing_api calls to return some JSON that has browse pages, orgs, etc... 

There are test helpers in the gds-api-adapters gem for stubbing the API calls and providing canned responses: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb and probably there are example tests in this project that use them which you can use for inspiration.